sentry_sdk/integrations/celery.py

```diff
@@ -487,6 +480,7 @@ def _patch_beat_apply_entry():
     # type: () -> None
     original_apply_entry = Scheduler.apply_entry
 
+    @ensure_integration_enabled(CeleryIntegration, original_apply_entry)
     def sentry_apply_entry(*args, **kwargs):
```

Comment:
We'll need to do this for the new redbeat wrapper too once we update `sentry-sdk-2.0` with current `master`. (Just a note for later.)